src/Motor.Extensions.Diagnostics.Queue.Metrics/Motor.Extensions.Diagnostics.Queue.Metrics.csproj

```diff
@@ -6,7 +6,7 @@
 
     <ItemGroup>
       <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.0.0" />
-      <PackageReference Include="Prometheus.Client" Version="5.0.0" />
+      <PackageReference Include="Prometheus.Client" Version="5.2.0" />
```

Comment:
`Prometheus.Client` depends on `Prometheus.Client.Abstractions` so having `Prometheus.Client.Abstractions` defined here as a dependency in the line below isn't necessary. I think we should remove it to prevent this issue from occurring again.